source

```diff
@@ -103133,23 +103133,28 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
    <dd>Switch to the <span>before DOCTYPE public identifier state</span>.</dd>
 
    <dt>U+0022 QUOTATION MARK (&quot;)</dt>
-   <dd><span>Parse error</span>. Set the DOCTYPE token's public identifier to the empty string (not
+   <dd>This is a <dfn data-x="parse-error-missing-whitespace-after-doctype-public-keyword">missing-whitespace-after-doctype-public-keyword</dfn>
+   <span>parse error</span>. Set the DOCTYPE token's public identifier to the empty string (not
    missing), then switch to the <span>DOCTYPE public identifier (double-quoted) state</span>.</dd>
 
    <dt>U+0027 APOSTROPHE (')</dt>
-   <dd><span>Parse error</span>. Set the DOCTYPE token's public identifier to the empty string (not
+   <dd>This is a <span data-x="parse-error-missing-whitespace-after-doctype-public-keyword">missing-whitespace-after-doctype-public-keyword</span>
+   <span>parse error</span>. Set the DOCTYPE token's public identifier to the empty string (not
    missing), then switch to the <span>DOCTYPE public identifier (single-quoted) state</span>.</dd>
 
    <dt>U+003E GREATER-THAN SIGN (&gt;)</dt>
-   <dd><span>Parse error</span>. Set the DOCTYPE token's <i data-x="force-quirks flag">force-quirks flag</i> to <i>on</i>.
+   <dd>This is an <dfn data-x="parser-error-abrupt-closing-of-doctype">abrupt-closing-of-doctype</dfn>
+   <span>parse error</span>. Set the DOCTYPE token's <i data-x="force-quirks flag">force-quirks flag</i> to <i>on</i>.
    Switch to the <span>data state</span>. Emit that DOCTYPE token.</dd>
 
    <dt>EOF</dt>
-   <dd><span>Parse error</span>. Set the DOCTYPE token's <i data-x="force-quirks flag">force-quirks
+   <dd>This is an <span data-x="parse-error-eof-in-doctype">eof-in-doctype</span> <span>parse error</span>.
+   Set the DOCTYPE token's <i data-x="force-quirks flag">force-quirks
    flag</i> to <i>on</i>. Emit that DOCTYPE token. Emit an end-of-file token.</dd>
 
    <dt>Anything else</dt>
-   <dd><span>Parse error</span>. Set the DOCTYPE token's <i data-x="force-quirks flag">force-quirks flag</i> to <i>on</i>.
+   <dd>This is a <dfn data-x="parse-error-missing-quote-around-doctype-public-identifier">missing-quote-around-doctype-public-identifier</dfn>
```

Comment:
Sorry for the wrong suggestion, we can reuse `missing-quote-before-doctype-public-identifier` error here.